internal/pkg/api/handleCheckin.go

```diff
@@ -181,6 +181,13 @@ func (ct *CheckinT) validateRequest(zlog zerolog.Logger, w http.ResponseWriter,
 	}
 	cntCheckin.bodyIn.Add(readCounter.Count())
 
+	if req.Status == CheckinRequestStatus("") {
+		return val, fmt.Errorf("checkin status missing")
+	}
+	if len(req.Message) == 0 {
+		return val, fmt.Errorf("checkin message missing")
+	}
```

Comment:
On what cases these fields were empty? Is it possible that we are breaking checkin in some case?

Reply:
@cmacknz, would the agent ever try to check in with an empty message?

Reply:
It shouldn't, but if there were a bug that caused this, would it be better to accept the checkin instead of marking the agent offline and updating the component information?

Reply:
I've changed this to log a warning instead